elements/bulbs-video/bulbs-video.js

```diff
@@ -47,13 +79,15 @@ Object.assign(BulbsVideo, {
     controller: ControllerField,
   },
   propTypes: {
-    autoplay: PropTypes.bool,
-    autoplayNext: PropTypes.bool,
-    loop: PropTypes.bool,
-    mute: PropTypes.bool,
+    autoplay: PropTypes.string,
+    autoplayNext: PropTypes.string,
+    noEndcard: PropTypes.string,
     src: PropTypes.string.isRequired,
     twitterHandle: PropTypes.isRequired,
   },
 });
 
 registerReactElement('bulbs-video', BulbsVideo);
+
+import './elements/meta';
+import './elements/summary';
```

Comment:
Elements:

* registered as dom elements with `document.registerElement`
* have a special wrapper that marries `document.registerElement` with `ReactDOM.render`

Components
* mere react components (both classes and functions)